cmd/influx_tsm/converter.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/influxdata/influxdb/tsdb/engine/tsm1"
 )
 
+// KeyIterator is the interaface to iterate over b*1 keys for conversion to tsm1 keys
```

Comment:
Interface is spelled wrong. I also think you can change the wording here so you aren't saying "this is an interface". You can just say something like `KeyIterator is used to iterate over b* keys for conversion to tsm keys`.

Also avoid referring to `tsm` as `tsm1`. The version number isn't important.